functions/Get-DbaAgentJobHistory.ps1

```diff
@@ -32,6 +32,9 @@ function Get-DbaAgentJobHistory {
     .PARAMETER EndDate
         The DateTime before which the history is wanted. If unspecified, all available records will be processed.
 
+    .PARAMETER OutcomeType
```

Comment:
Please add an example for this parameter...

Reply:
Good point. Done.